EurorackPanelDesigner/INSTALL.txt

```diff
@@ -1,5 +1,25 @@
 Eurorack Panel Designer
 
-Unarchive and copy files to Inkscape's "extensions" directory (usually inkscape/share/extensions).
+Unarchive and copy files to Inkscape's "extensions" directory.
```

Comment:
Apparently this is recommended install dir instruction now, as guaranteed to be correct for every user.